src/Disks/IO/CachedOnDiskReadBufferFromFile.cpp

```diff
@@ -251,7 +250,7 @@ CachedOnDiskReadBufferFromFile::getReadBufferForFileSegment(FileSegmentPtr & fil
         }
         else
         {
-            LOG_DEBUG(log, "Bypassing cache because `read_from_filesystem_cache_if_exists_otherwise_bypass_cache` option is used");
+            LOG_TRACE(log, "Bypassing cache because `read_from_filesystem_cache_if_exists_otherwise_bypass_cache` option is used");
```

Comment:
I noticed that this message is printed too frequently? Maybe LOG_TEST?

Reply:
Ok